### ISSUE TITLE
environment.md compilation fails during build

### ISSUE THREAD

**Issue by stefanvds** [cirg-up/cilib-tutorial#3]
```[tut] compiling: /source/src/pages/step/environment.md
[tut] *** Error reported at /source/src/pages/step/environment.md:28
// <console>:28: error: too many arguments (3) for method apply: (cmp: cilib.Comparison, eval: cilib.RVar[scalaz.NonEmptyList[A] => cilib.Objective[A]])cilib.Environment[A] in object Environment
// Note that 'bounds' is not a parameter name of the invoked method.
//            bounds = Interval(-5.12,5.12)^2
//                   ^
// <console>:16: warning: Unused import
//        import Scalaz._
//                      ^
// <console>:18: warning: Unused import
//        import spire.math.Interval
//                          ^
// <console>:20: warning: Unused import
//        import spire.implicits.{eu=>_, _}
//                                       ^
[error] (run-main-5) tut.TutException: Tut execution failed.```

**Comment by gpampara** [comment]
The last tagged PDF was built using a prerelease of cilib. The released version no longer maintains the `bounds` member in the `Environment`.

This may change going forward, but that's not a discussion to have here in this repository.

**Comment by gpampara** [comment]
Fixed with fdf8631
